src/cairoUtilFuncGen/event.ts

```diff
@@ -188,13 +185,13 @@ export class EventFunction extends StringIndexedFuncGen {
 
     this.requireImport('starkware.cairo.common.uint256', 'Uint256');
     this.requireImport(`warplib.maths.utils`, 'felt_to_uint256');
-    this.requireImport('warplib.keccak', 'warp_keccak_felt');
+    this.requireImport('warplib.keccak', 'warp_keccak');
     this.requireImport('warplib.dynamic_arrays_util', 'fixed_bytes256_to_felt_dynamic_array_spl');
 
     return [
       `   let (mem_encode: felt) = ${abiFunc}(${argName});`,
-      `   let (keccak_hash: felt) = warp_keccak_felt(mem_encode);`,
-      `   let (keccak_hash256: Uint256) = felt_to_uint256(keccak_hash);`,
+      `   let (keccak_hash256: Uint256) = warp_keccak(mem_encode);`,
+      // `   let (keccak_hash256: Uint256) = felt_to_uint256(keccak_hash);`,
```

Comment:
Why is it commented?

Reply:
This comment is redundant, lemme remove it